### PR TITLE
QuantityValue display in grid

### PR DESCRIPTION
**Pull request by IgorBenko**
Fixes the display of quantity value field in the grid. Before only this was displayed: [Object object]

### CODE REVIEW

File: pimcore/static6/js/pimcore/object/tags/quantityValue.js [hunk 1]
```diff
@@ -109,6 +109,35 @@ pimcore.object.tags.quantityValue = Class.create(pimcore.object.tags.abstract, {
...
+
+            if (value) {
+                return value.value + " " + value.unit;
+            } else {
+                return "";
+            }
+
+        }.bind(this, field.key);
+
+        console.log(field);
```
Comment (brusch) [inline pimcore/static6/js/pimcore/object/tags/quantityValue.js]: I think this console.log() shouldn't be there anymore, right? :)

Comment (IgorBenko) [inline pimcore/static6/js/pimcore/object/tags/quantityValue.js]: Whoops, of course not :) I will do a new pull request!

Comment (brusch) [inline pimcore/static6/js/pimcore/object/tags/quantityValue.js]: Merci! :)

**Comment by weisswurstkanone** [comment]
What happens if the unit does not exist anymore? As far as I remember there is no instant cleanup.

**Comment by IgorBenko** [comment]
@weisswurstkanone Good point. I will add a check for that!

**Comment by IgorBenko** [comment]
@brusch @weisswurstkanone It should be OK now :)

**Comment by brusch** [comment]
Thanks!

**Comment by IgorBenko** [comment]
Thank you! :)

**Pull request merged**